sqrl-base/src/main/java/com/datasqrl/config/SystemBuiltInConnectors.java

```diff
@@ -13,7 +13,8 @@ public enum SystemBuiltInConnectors {
   PRINT_SINK(Name.system("print"), ExternalDataType.sink),
   LOCAL_FILE_SOURCE(Name.system("localfile"), ExternalDataType.source),
   LOGGER(Name.system("logger"), ExternalDataType.sink),
-  LOG_ENGINE(Name.system("log"), ExternalDataType.sink);
+  LOG_ENGINE(Name.system("log"), ExternalDataType.sink),
+  ;
```

Comment:
Theres an extra `,` here.

Reply:
Yeah, I usually add one so it helps reduce the number of diff lines